### PR TITLE
Add console postsubmits job

### LINKED ISSUE #16: add postsubmit job for console
ref https://github.com/kubesphere/test-infra/pull/11
we only add test and building jobs for console, we still need to add postsubmit(e.g building and push images) jobs.
/assign

### PR DESCRIPTION
**Pull request by yuswift**
Signed-off-by: yuswift <yuswift2018@gmail.com>
Fixes #16 
1. Add a postsubmits job that builds images and push images
2. Add a daily build job.

### CODE REVIEW

File: config/jobs/kubesphere/console/console-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+postsubmits:
+  kubesphere/console:
+  - name: pull-console-build-image
+    labels:
+      preset-docker-sock: "true"
+      preset-docker-credential: "true"
```
Comment (zryfish) [inline config/jobs/kubesphere/console/console-postsubmits.yaml:6]: It's `preset-docker-credentials`, https://github.com/kubesphere/test-infra/blob/main/config/prow/config.yaml#L141

File: config/jobs/kubesphere/console/console-postsubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  labels:
+    preset-docker-sock: "true"
+    preset-docker-credential: "true"
+  spec:
+    containers:
+    - image: kubesphere/build-tools:master-latest
+      command:
+      - "/bin/sh"
+      - "-c"
+      - TAG=nightly-$(date "+%Y%m%d") make image && TAG=nightly-$(date "+%Y%m%d") make image-push
```
Comment (zryfish) [inline config/jobs/kubesphere/console/console-postsubmits.yaml:42]: Why don't we make TAG an env, make it globally across the whole build? Imagine `make image` happens at 23:59, and it took 10 mins to finish, so `make image-push` will have a different TAG with `make image`

Comment (yuswift) [reply config/jobs/kubesphere/console/console-postsubmits.yaml]: Nice catch. Done. `TAG=xxx` -> `export TAG`

File: config/jobs/kubesphere/console/console-postsubmits.yaml (hunk 1 above)
Comment (yuswift) [reply config/jobs/kubesphere/console/console-postsubmits.yaml]: Done

File: config/jobs/kubesphere/console/console-postsubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  decorate: true
+  branches: master
+  extra_refs:
+  - org: kubesphere
+    repo: console
+    base_ref: master
+  name: kubesphere-console-periodic-build-image
+  labels:
+    preset-docker-sock: "true"
+    preset-docker-credential: "true"
```
Comment (zryfish) [inline config/jobs/kubesphere/console/console-postsubmits.yaml:35]: Forgot here

Comment (yuswift) [reply config/jobs/kubesphere/console/console-postsubmits.yaml]: done

**Pull request merged**